test/test_market_makers.js

```diff
@@ -161,8 +172,8 @@ contract('MarketMaker', function(accounts) {
                 )
             }
 
-            // Price is equal to 1
-            assert.equal(cost, tokenCount)
+            // Price is at least 1
+            assert(cost.gte(tokenCount))
```

Comment:
This change was made because of the rounding compensation in the old code causing the market maker to price the outcome tokens more than collateral sometimes. This changeset actually permits the stricter equals check.